tests/xfstests/run.pm

```diff
@@ -153,15 +61,11 @@ sub test_name {
 # status - test status
 # time   - time consumed
 sub log_add {
-    my ($file, $test, $status, $time) = @_;
+    my ($test, $status, $time) = @_;
     my $name = test_name($test);
-    unless ($name and $status) { return; }
-    my $cmd = "echo '$name ... ... $status (${time}s)' >> $file && sync $file";
-    send_key 'ret';
-    assert_script_run($cmd);
-    sleep 5;
-    my $ret = script_output("cat $file", 20);
-    return $ret;
+    die "Required variable test: $test or status: $status is missing" unless ($name and $status);
+    type_string("\n");
```

Comment:
what's wrong with the previous `send_key 'ret'`?

Reply:
This PR is about using serial console, try to ask again ......

Reply:
ah, because send_key does not work on serial_terminal? Well, anyway, it's ok then